install_git.ps1

```diff
@@ -0,0 +1,39 @@
+Function global:ADD-PATH()

```

Comment:
There's a really cool Powershell standard, you should try to follow it: https://github.com/PoshCode/PowerShellPracticeAndStyle/blob/master/Style%20Guide/English.md